ecosystem/sep-0012.md

```diff
@@ -48,6 +49,82 @@ Alternatively, if the client cannot add the authorization header. The JWT should
 ?jwt=<token>
 ```
 
+## Customer GET
+
+Checks the status of a customers information. This allows clients to check whether the customers information was accepted, rejected, or still needs more info.  Lookups are done via the stellar account ID and memo that were used for the initial PUT request.
+
+This also gives you the information needed as far as what information a KYC provider requires to KYC a user.  If you fetch an account that has never been KYC'd the server should respond with `NEEDS_INFO` and a list of all the fields you will need to provide.
+
+### Request
+
+```
+GET [KYC_SERVER]/customer?stellar_account_id=<public_key>&memo=<memo>
+```
+
+Name | Type | Description
+-----|------|------------
+`account` | `G...` string | The Stellar account ID used to identify this user
+`memo` | string | (optional) Unique identifier for user being identified, necessary for custodial, multi-user accounts.
+`memo_type` | string | (optional) type of `memo`. One of `text`, `id` or `hash`
+`lang` | string | (optional) Defaults to `en`.  Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1).  Human readable descriptions and reasons should be in this language.
+
+### Response
+
+Name | Type | Description
+-----|------|------------
+`status` | string | Status of the customers KYC process.
+`fields` | array<object> | An array of fields still needed to finish KYC Process
+`reason` | string | Human readable reason for rejection
+
+```js
+// The case when a user has been successfully KYC'd and approved
+{
+   "status": "ACCEPTED",
+}
+```
+
+```js
+// The case when an anchor requires more info a bout a user
+{
+   "status": "NEEDS_INFO",
+   "fields": [
+      {
+         "name": "email_address",
+         "description": "Email address of the user",
+         "type": "string"
+      }
+   ]
+}
+```
+
+```js
+// The case when a user has been rejected and cannot be KYC'd
+{
+   "status": "REJECTED",
+   "reason": "This person is on a sanctions list"
+}
+```
+
+#### Status
+
+Status | Description
+-------|------------
+ACCEPTED | KYC has been accepted and user can be identified by stellar account and optional memo in future transactions
+PROCESSING | KYC process is in flight and client can check again in the future to see if any further info is needed
+NEEDS_INFO | More info needs to be provided to finish KYC for this user.  The `fields` entry is required in this case.
```

Comment:
We should make a comment about how an originally `ACCEPTED` user can move to `NEEDS_INFO` and `PROCESSING` again if the anchor realizes more info is necessary.